var/spack/repos/builtin/packages/ucx/package.py

```diff
@@ -51,6 +54,8 @@ class Ucx(AutotoolsPackage, CudaPackage):
             description='Enable logging')
     variant('debug', default=False,
             description='Enable debugging')
+    variant('opt', default='0', values=('0', '1', '2', '3'), multi=False,
```

Comment:
why 0?

Reply:
Yup, did not read the code correctly. It should be 3.